cmd/zc_traverser_local_test.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"os"
+
+	chk "gopkg.in/check.v1"
+)
+
+type localTraverserTestSuite struct{}
+
+var _ = chk.Suite(&localTraverserTestSuite{})
+
+func (s *localTraverserTestSuite) TestCleanLocalPath(c *chk.C) {
+	testCases := map[string]string{
+		"/user/foo/bar":    "/user/foo/bar", // regular unix path with no change
+		"/user/foo/bar/":   "/user/foo/bar", // regular unix path with extra slash
+		"/user//foo//bar/": "/user/foo/bar", // regular unix path with double slashes
+		"./foo/bar":        "foo/bar",       // relative unix path
+		"../foo/bar":       "../foo/bar",    // relative unix path with parent dir
+		"foo/bar":          "foo/bar",       // shorthand relative unix path
+	}
+
+	for orig, expected := range testCases {
+		c.Assert(cleanLocalPath(orig), chk.Equals, expected)
+	}
+}
+
+func (s *localTraverserTestSuite) TestCleanLocalPathForWindows(c *chk.C) {
+	// ignore these tests when not running on Windows
+	// as the path cleaning behavior depends on the platform
+	if os.PathSeparator != '\\' {
+		c.SucceedNow()
```

Comment:
Another quick note: Use `c.Skip(reason string)` instead. It'll give a more clear indication of why this test was not run.